src/components/forms/DatePicker/DatePicker.test.tsx

```diff
@@ -21,13 +21,32 @@ describe('DatePicker component', () => {
     name: 'birthdate',
   }
 
-  it('renders witout errors', () => {
-    const { getByTestId } = render(<DatePicker {...testProps} />)
-    expect(getByTestId('date-picker')).toBeInTheDocument()
+  const renderDatePicker = (
+    props?: Omit<ComponentProps<typeof DatePicker>, 'id' | 'name'>
+  ) => {
+    const allProps = {
+      ...testProps,
+      ...props
+    }
+    const { getByLabelText, getByTestId, getByText } = render(<DatePicker {...allProps} />)
+    const queryForDatePicker = () => screen.queryByTestId('date-picker')
+    return {
+      getByLabelText,
+      getByTestId,
+      getByText,
+      queryForDatePicker,
+    }
+  }
```

Comment:
```suggestion
    const queryForDatePicker = () => screen.queryByTestId('date-picker')
    return {
      ...rendered,
      queryForDatePicker,
    }
```

Reply:
Just a shorthand to expose all the things from the `render` method

Reply:
TIL thanks!